_monitoring-plugins/alerting/monitors.md

```diff
@@ -23,76 +23,7 @@ Monitor | A job that runs on a defined schedule and queries OpenSearch indices.
 Trigger | Conditions that, if met, generate *alerts*.
 Alert | An event associated with a trigger. When an alert is created, the trigger performs *actions*, which can include sending a notification.
 Action | The information that you want the monitor to send out after being triggered. Actions have a *destination*, a message subject, and a message body.
-Destination | A reusable location for an action. Supported locations are Amazon Chime, Email, Slack, or custom webhook.
```

Comment:
As a general comment, I think we could add a little FAQ section somewhere in the Alerting documentation describing that Destinations are deprecated and the Notifications integration.

This is a rough idea of what I had in mind, feel free to reword it and add it as its own section and let me know if you'd like any other questions answered here:

### Alerting Notifications FAQ

* What plugins do I need installed besides Alerting?
To continue using the notification Action in Alerting you'll want to have the `notifications-core` and `notifications` backend plugins installed. The Notification Dashboards plugin can also be installed for managing the channels via OpenSearch Dashboards.

* Can I still create Destinations?
No, Destinations have been deprecated and can no longer be created/edited.

* Will I need to move my Destinations to Notifications?
No, for upgrading users, a background process will automatically migrate Destinations to Notification Channels. These Channels will have the same ID as the Destinations and the Monitor execution will resolve to the correct one on behalf of the user so no changes will need to be made to the Monitor definition either. The migrated Destinations will be deleted.

* What happens if any Destinations fail to migrate?
If a Destination failed to migrate, the Monitor execution will continue to use it until it has migrated to a Notification Channel. No user intervention is required in this case.

* Do I need the Notification plugins installed if Monitors can still use Destinations?
Yes, the fallback on Destination is to prevent failure in sending messages when the migration fails. However, the Notification plugin is what actually sends the message. Not having the Notification plugin installed will lead to the Action failing.

Reply:
Added